### PR TITLE
Addendum to 8 GitHub prep.md

### LINKED ISSUE #49: Edit to token generation section of 8-Github-Prep.md
Within the section "I have the URL but have yet to connect to GitHub", the file explains how to generate a token, but appears to leave out the crucial point that the generated token will no longer be viewable after the user has left the generation page, which was an issue that I myself encountered. I believe that including this detail, likely bolded, will better ensure that the user is more careful of how they go about securing their token.

### PR DESCRIPTION
**Pull request by bayers1**
Fixes #49 
(Accidentally closed the previous pull request.)

### CODE REVIEW

**Review by onzfonz — CHANGES_REQUESTED** [review]
Please separate based on sentences so that new sentences start on a new line.
Please also make sure that you put fixes # and your issue in the pull request description @bayers1

**Comment by bayers1** [comment]
Sentences have been readjusted and the issue is now listed.

**Comment by onzfonz** [comment]
you need to remove the word issue from the description

File: docs/labs/8-Github-Prep.md [hunk 1]
```diff
@@ -136,6 +136,9 @@ and then go to *Settings->Developer settings->Personal Access Token->Create New
 Then give the token a name,
 set expiration to *No expiration* or *90 days* and select the ```all repos``` checkbox at the top.
 This will be used in place of your password.
+Be sure that you have saved this token, as once you leave the generation page, ***the token
+will no longer be visible.***
```
Comment (onzfonz) [inline docs/labs/8-Github-Prep.md:140]:
```suggestion
Be sure that you have saved this token, as once you leave the generation page,
***the token will no longer be visible***.
```

**Review by onzfonz — CHANGES_REQUESTED** [review]
please commit via commit suggestions

**Pull request closed without merging**